setup.py

```diff
@@ -16,13 +10,13 @@
     url='http://github.com/readthedocs/readthedocs-sphinx-ext',
     license='MIT',
     description='Sphinx extension for Read the Docs overrides',
+    install_requires=['requests', 'Jinja2>=2.9'],
     package_dir={'': '.'},
-    packages=find_packages('.'),
+    packages=find_packages('.', exclude=['tests']),
```

Comment:
Not sure about this, but I think some distributions run the tests after installing a package, so they expect them to be included

Reply:
Distributions usually run tests *before* building the package. According to my testing, `setup.py sdist` still includes them in the tarball. If you want them to be usable after installing the package, they need to be moved inside the package. Installing a global scope `tests` package is a very bad idea, and guaranteed to cause problems when done by more than one package.

Reply:
That makes sense to me, I also saw https://github.com/readthedocs/readthedocs-sphinx-ext/pull/60